thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -370,11 +370,11 @@ protected AnomalyFunctionDTO getDefaultFunctionSpecByTimeGranularityLegacy(TimeG
         anomalyFunctionSpec.setRequiresCompletenessCheck(false);
         break;
       case DAYS:
-        anomalyFunctionSpec.setType("SPLINE_REGRESSION_VANILLA");
+        anomalyFunctionSpec.setType("SPLINE_REGRESSION_WRAPPER");
         anomalyFunctionSpec.setCron("0 0 14 * * ? *");
         anomalyFunctionSpec.setWindowSize(1);
         anomalyFunctionSpec.setWindowUnit(TimeUnit.DAYS);
-        anomalyFunctionSpec.setProperties("");
+        anomalyFunctionSpec.setProperties("variables.continuumOffset=P90D;module.training=parametric.GenericSplineTrainingModule;variables.numberOfKnots=0;variables.degree=3;variables.predictionMode=TRENDING;variables.anomalyRemovalThreshold=0.6,-0.6;module.data=ContinuumDataModule;variables.pValueThreshold=0.025;function=ConfigurableAnomalyDetectionFunction;variables.seasonalities=DAILY_SEASONALITY;module.detection=ConfidenceIntervalDetectionModule;module.testingPreprocessors=DummyPreprocessModule;workflow=RegressionWorkflow;variables.recentPeriod=P14D;module.trainingPreprocessors=AnomalyRemovalByWeight;variables.r2Cutoff=0.9");
```

Comment:
Please add "downgrade.variables.seasonalities=" into the properties.

Reply:
Thanks. Added. Please also let TE aware to modify the onboarded alerts using SplineRegressionWrapper on the additional properties.